app/styles/templates/components/_package_summary_block.scss

```diff
@@ -1,4 +1,4 @@
-.move-item-to-box, .package-summary-block {
+.package-summary-block {
```

Comment:
Make sure to test UI in add box-pallet overlay where this CSS is used.

Reply:
I did, It's just a css class renaming